fronts-client/src/components/feed/RecipeSearchContainer.tsx

```diff
@@ -144,11 +180,99 @@ export const RecipeSearchContainer = ({ rightHandContainer }: Props) => {
 							setPage(1);
 							setSearchText(event.target.value);
 						}}
+						onClick={() => setShowAdvancedRecipes(true)}
 						value={searchText}
 					/>
 				</TextInputContainer>
 				<ClipboardHeader />
 			</InputContainer>
+
+			{showAdvancedRecipes && selectedOption === FeedType.recipes ? (
+				<>
+					<TopOptions>
+						<div>
+							<label
+								htmlFor="dateSelector"
+								style={{ color: searchText == '' ? 'gray' : 'inherit' }}
+							>
+								Ordering priority
+							</label>
+						</div>
+						<div>
+							<select
+								id="dateSelector"
+								value={dateField}
+								disabled={searchText == ''}
+								onChange={(evt) => {
+									console.log(evt.target);
+									setDateField(
+										evt.target.value === 'Relevance'
+											? undefined
+											: (evt.target.value as DateParamField),
+									);
+								}}
+							>
+								<option value={'Relevance'}>
```

Comment:
Using an uppercase R here feels inconsistent. I suppose it’s signalling that this option isn’t a field name? I think I’d still prefer lowercase.